### DISCUSSION TITLE
Create a mode in which it periodically takes the transcription text that is already ready at the moment and displays it in subtitles

### DISCUSSION THREAD

**Discussion by LebedevIV** [SubtitleEdit/subtitleedit#7995] [Q&A]
Is it possible for Whisper (and other models) to create a mode in which it periodically takes the transcription text that is already ready at the moment and displays it in subtitles? Now, you either have to wait for the end of processing, which can take quite a long time, or interrupt processing. This is necessary to control the quality of the transcription so as not to rely blindly, waiting for a long time for the result, but to immediately see “online” what happens.

**Comment by sherkhan30452** [comment]
If you mean dynamic subtitles, I suggest you this software called Speech Translate which creates subtitles like youtube auto-generate, you can change the parameters to suit your needs. 

Make sure to download correct version for your hardware, it has two setups - GPU and CPU only.

Hope this helps.

https://github.com/Dadangdut33/Speech-Translate
